oak_functions/lookup_data_generator/src/data.rs

```diff
@@ -111,15 +111,19 @@ pub fn generate_and_serialize_sparse_weather_entries<R: Rng>(
     let lat_dist = rand::distributions::Uniform::new(-1.0_f64, 1.0);
     let mut cell_map = MultiMap::new();
     for i in 0..entries {
-        // Fix the first point to ensure the default example can find at least one data point within
-        // 40km.
+        // Fix the first and the second point to ensure the default example can find at least one
+        // data point within 40km.
         let latitude_degrees = if i == 0 {
             52.1_f64
```

Comment:
Where are these values coming from? Could you add links to the things that actually need them? Also reverse links from the example to this file. Otherwise there is nothing to keep them in sync apart from trying it out and see it fail with an unrelated error message.

Reply:
Done